content/_changelogs/9.6.0.md

```diff
@@ -0,0 +1,55 @@
+## 9.6.0
+
+_Released 4/25/2022_
+
+**Features:**
+
+- Now you can easily test multi-domain workflows by using the experimental
+  [`cy.origin()`](/api/commands/origin) command. This feature allows you to test
+  across different origins or domains within a single test. This experiment was
+  designed to work side-by-side with the [`cy.session()`](/api/commands/session)
+  experimental feature. You can turn on use of this command by setting the
+  [`experimentalSessionAndOrigin`](guides/references/experiments) configuration
+  option to `true`. For more details, read
+  [our blog post](https://cypress.io/blog/2022/04/25/cypress-9-6-0-easily-test-multi-domain-workflows-with-cy-origin/).
+  Addressed [#17336](https://github.com/cypress-io/cypress/issues/17336) and
+  [#944](https://github.com/cypress-io/cypress/issues/944).
+  - Added a new configuration option called `experimentalSessionAndOrigin`. This
+    option enables support for [`cy.session()`](/api/commands/session) and
+    [`cy.origin()`](/api/commands/origin).
+  - The `experimentalSessionSupport` configuration option has been removed and
+    replaced by the `experimentalSessionAndOrigin` configuration option. Please
+    update your configuration to use the new `experimentalSessionAndOrigin`
+    option to continue to opt-in to this experimental feature.
+  - When `experimentalSessionAndOrigin` is enabled, cross-origin requests will
+    no longer immediately fail, but instead, time out based on the
+    [`pageLoadTimeout`](/guides/references/configuration#Timeouts)
+    configuration. This allows the request to be appropriately handled by the
+    [`cy.origin()`](/api/commands/origin) command and if it isn't handled, it's
+    assumed it was an unexpected request that should fail.
```

Comment:
I don't think this explanation is needed in the changelog.

```suggestion
    configuration.
```

Reply:
Typically in the changelog we provide the previous behavior on why a change was needed or what it improving. I think standalone calling this change out doesn't make sense why the change was needed. What if I slim to ```suggestion
    configuration. This allows the request to be appropriately handled by the
    [`cy.origin()`](/api/commands/origin) command.
```